.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules/
+node_modules/
```

Comment:
If `node_modules` is now ignored, does it make sense to remove the folder as well?